packages/select/src/vaadin-select.js

```diff
@@ -627,6 +643,49 @@ class Select extends OverlayClassMixin(
     itemElement.setAttribute('id', this._itemId);
   }
 
+  /**
+   * @param {string} accessibleName
+   * @private
+   */
+  _accessibleNameChanged(accessibleName) {
+    this._srLabelController.setLabel(accessibleName);
+    this._setCustomAriaLabelledBy(accessibleName ? this._srLabelController.defaultId : null);
+  }
+
+  /**
+   * @param {string} accessibleNameRef
+   * @private
```

Comment:
Same as above, please change this method to be `@protected` in the mixin.

Reply:
Done.